src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPage.cs

```diff
@@ -82,15 +82,15 @@ protected bool IsDirty
         /// </summary>
         private T WaitForAsync<T>(Func<Task<T>> asyncFunc)
         {
-            return _threadHandling.ExecuteSynchronously<T>(asyncFunc);
+            return _threadHandling != null ? _threadHandling.ExecuteSynchronously<T>(asyncFunc) : default(T);
```

Comment:
There's a (very small) comment mentioning one case on line 347. I've hit it multiple times closing the property pages.